generators/languages/index.js

```diff
@@ -78,69 +80,80 @@ module.exports = class extends BaseGenerator {
                 }
             });
         }
+        const blueprint = this.options.blueprint || this.configOptions.blueprint || this.config.get('blueprint');
+        // use global variable since getters dont have access to instance property
+        if (!opts.fromBlueprint) {
+            useBlueprint = this.composeBlueprint(blueprint, 'languages', {
+                'skip-install': this.options['skip-install'],
```

Comment:
I'm pretty sure it's best to pass the same options to the blueprint subgenerator as the ones the jhipster subgenerator has received: https://github.com/jhipster/generator-jhipster/blob/master/generators/generator-base.js#L2691
So I would also add: `'skip-server'`, `'skip-client'`, `langages` and probably `debug` too.

Reply:
thx, it's done